content/en/docs/tasks/traffic-management/locality-load-balancing/_index.md

```diff
@@ -46,9 +46,9 @@ Localities are hierarchical, in the matching order:
 
 1. Region
 
-1. Zone
+2. Zone
```

Comment:
The markdown can do math. This changes is not necessary and the original is better IMO.